README.md

```diff
@@ -20,6 +20,7 @@ Each package has its own **node_modules** directory that contains symbolic links
 
 - [Git](https://git-scm.com/)
 - [Node](https://nodejs.org/en/): an installation of the latest security patch of Node 20. The Node installation also includes the **npm** package manager.
+- [pnpm](https://pnpm.io/installation#using-npm); `pnpm` is our preferred package manager, it is more performant and monorepo friendly than `npm`. To install, do `npm install -g pnpm`
```

Comment:
i dont recommend installing pnpm via npm. use corepack

Reply:
updated readme - we should be aware the instructions might change in the future depending on what the node maintainers [decide to do](https://medium.com/@pacoarjona/node-js-leaves-npm-55d022e95a4f). A later problem though

Reply:
In other words, I feel that "preferred" is incorrect and should be "required". It's true that you can use npm to run scripts from package.json files in the various subdirectories, and I guess pnpm (or better yet rushx?) might be "preferred" for doing that.

Reply:
That's true, I should change it to `required` - as for preference, it was my intention to update this readme to recommend that people **do** use `pnpm` to run scripts within subdirectories, I think I'll find a spot in the readme to explicitly say that

Reply:
I definitely don't have a problem with indicating that pnpm is preferred (vs. required) over npm when running specific scripts in subdirectories. This line implies to me that pnpm is only preferred and not required for the repository as a whole, since this is the top-level README.md.

Reply:
I've updated this section to point out `pnpm` is required, and added a quote block recommending people to use `pnpm` over `npm` when running scripts within subdirectories

Reply:
Looks good. I won't resolve this conversation, since I didn't start it.

Reply:
> i dont recommend installing pnpm via npm. use corepack

Before @aruniverse's recommendation, I installed with "npm install -g pnpm". This works fine. Then I tried "corepack enable pnpm" but this gave an error unless I used an administrator shell. Also I'm not sure this did anything since I already have pnpm. So:
* Is "corepack enable pnpm" all one needs to do to get pnpm the recommended way?
* If so, maybe just say that instead of providing the link?
* And maybe add a note about admin privileges.

Reply:
I've edited the README according to your 2 bullet points dave, the third one I didn't see anything new added to the top package.json... I did run npm uninstall -g pnpm before doing it, but no dice

Reply:
Did your next "pnpm test" install pnpm? That's when the "packageManager" line got appended to package.json. Or maybe you already had that line. It did inform me that I lacked such a line, and so it was adding it.

Reply:
Yes, I ran pnpm test after running corepack. The thing is, we shouldn't need a packageManager property to be appended to package.json, not at the root level for sure, since our rush.json has a [property](https://github.com/iTwin/itwinjs-core/blob/52e966a8b3ddaafa57315a06755d14ffb22b0d65/rush.json#L4) that tells the whole repo to use pnpm as a package manager